src/main/steps/c100-rebuild/miam/mediator-document/template.njk

```diff
@@ -0,0 +1,7 @@
+{% extends "common/template.njk" %}
+
+
+{% block page_content %}
+<span class="govuk-caption-xl">{{caption}}</span>
```

Comment:
@monalisasarkar1 Have you defined caption in content.ts ?
If not then please remove this unused line.

Reply:
done